src/sdk/resources/configuration.ts

```diff
@@ -0,0 +1,16 @@
+import { IHttpClient } from '../http';
+import { BaseResource } from './resource';
+
+export class ConfigurationResource extends BaseResource {
```

Comment:
Direct access to modify the config endpoints will not be available once an application is in `main` stage. We will likely want a dedicated endpoint for accepting the EULA